sirepo/package_data/template/srw/parameters.py.jinja

```diff
@@ -437,6 +437,15 @@ def epilogue():
     pass
     {% endif %}
 
+{% if isBeamlineAnimation %}
+def process_watch(**kwargs):
+    {% if in_server %}
+    import sirepo.template
+    sirepo.template.run_on_server("srw", "process_watch", **kwargs)
```

Comment:
This is too much code. We don't need to dispatch. I don't think there needs to be a separate function. See below.